### ISSUE TITLE
There is no Russian version of this site

### ISSUE THREAD

**Issue by usersavchenko** [webpack/webpack.js.org#4886]
Is there any opportunity to translate manually English documentation to a different language ?
It gonna be cool to have such one. I would be able to add it.

@znarf  @jspears @eventualbuddha @kswedberg @Vinnl

**Comment by chenxsan** [comment]
Currently we have only a Chinese version translated by a group of volunteers from China https://github.com/docschina/webpack.js.org, and the website is hosted on their own server and domain https://webpack.docschina.org/. As far as I can see, we don't have enough resource to maintain translations in this repository. I think you can fork it and translate as you wish. When the site is ready, let us know.

**Comment by chenxsan** [comment]
I'm closing it at the moment, feel free to file a new issue once the site is ready.

**Comment by usersavchenko** [comment]
Okay
